pkg/reconciler/route/traffic/traffic.go

```diff
@@ -107,12 +104,16 @@ func (t *Config) GetRevisionTrafficTargets(domain string) []v1alpha1.TrafficTarg
 				LatestRevision: tt.LatestRevision,
 			},
 		}
-		if tt.Tag != "" && domain != "" {
+		if tt.Tag != "" {
 			// http is currently the only supported scheme
-			results[i].URL = TagURL(HTTPScheme, tt.Tag, domain)
+			fullDomain, err := domains.DomainNameFromTemplate(ctx, r, domains.SubdomainName(r, tt.Tag))
+			if err != nil {
+				return nil, err
+			}
+			results[i].URL = domains.URL(HTTPScheme, fullDomain)
```

Comment:
Is this a different `HTTPScheme` constant?

Reply:
Whoops! That was supposed to be destroyed.